packages/eas-cli/src/commands/build/view.ts

```diff
@@ -47,7 +44,11 @@ export default class BuildView extends EasCommand {
       if (buildId) {
         build = await BuildQuery.byIdAsync(buildId);
       } else {
-        const builds = await BuildQuery.allForAppAsync(projectId, { limit: 1 });
+        const builds = await BuildQuery.viewBuildsOnAppAsync({
+          appId: projectId,
+          offset: 0,
```

Comment:
I don't feel super strongly, but here is my take:
- I think the gql queries themselves should not have fallbacks. Make the lowest level methods explicit to avoid ambiguity/misunderstands. Add wrappers around them for convenience.
- The other gql queries I've touched so far have `$offset: Int!` so it's a standard I'm building here

If you feel strongly I'm happy to make the change / add a follow up task to change the other gql query params